samples/sample_monthly_update.md

```diff
@@ -0,0 +1,70 @@
+# Sample Update
+
+We ask our companies to send us regular updates so that we can stay up to date with the company, and help out as necessary. To help us help you, here is a sample of what we need to see.
+
+We ask that you spend at most 15-30 minutes filling out this information, as you will want systems in place to easily track this data.
+
+## Outline
+
+* **Our Victories**
+* **Our Struggles**
+* **Key Metrics**
+* **How You Can Help**
+* **Other Notes**
+
+---
+
+## Teacher Tools Inc. - May Update
+
+We're excited to move into June after a great school year. We've had our third consequetive month of growth and we've hired two new developers. Unfortunately our spending is a little higher than expected and we're adjusting our plans.
+
+### Our Victories
+
+1. We've grown our customer base in Ontario and Québec, and we're on track to meet our next milestones.
+2. Our team spent the last iteration on adding the new dashboard and it shipped on June 8th.
+
+### Our Struggles
+
+1. We're spending an aggressive amount on ads in order to acquire customers.
+2. We had mixed feedback on our dashboard so far. We were expecting a more positive response.
+3. Our dashboard took three weeks to ship instead of one. We underestimated how complex it would be.
+
+### Key Metrics
+
+#### Business and Financial
+
+1. **User Count**: 2,500
+2. **Customer Count (Paying)**: 1,315
+3. **Customer Lifetime Value (CLTV)**: $270.00
+4. **Customer Acquisition Cost (CAC)**: $233.00
+5. **Monthly Recurring Revenue (MRR)**: $19,725 (1315 * $15.00/mo)
+6. **MRR Growth Month-over-Month (CMGR)**: 19.54%
+7. **Burn Rate**: $17,390/mo
+8. **Remaining Runway**: 7 months.
+
+#### Product & Engagement
+
+1. **Monthly Active Users (MAU)**: 805
+2. **MAU Growth Month-over-Month**: 25.32%
+3. **Monthly Active Customers (MAC)**: 253
+4. **MAC Growth Month-over-Month**: 19.83%
+5. **Monthly Churn**: 8.35%
+
+#### Retention
+
+*Note: This is a sample set of images, but they should match your data*
+
+![Sample Weekly Retention](sample_weekly_retention.png)
```

Comment:
I used some sample images from Amplitude, but struggled to get quality ones. Short of faking months of data for a sample image, I figured this was best.